access-control-app/int_test/cmr/access_control/int_test/acl_crud_test.clj

```diff
@@ -1018,3 +1018,117 @@
                                    (u/conn-context) acl-concept-id {:token token :raw? true})]
         (is (= 401 status))
         (is (= ["Permission to delete ACL is denied"] (:errors body)))))))
+
+(deftest entry-titles-concept-ids-sync
+  (let [token (e/login-guest (u/conn-context))
+        make-catalog-item (fn [name coll-id]
+                            (-> catalog-item-acl
+                                (assoc-in [:catalog_item_identity :name] name)
+                                (assoc-in [:catalog_item_identity :collection_identifier] coll-id)))
+        coll1 (u/save-collection {:entry-title "coll1 entry title"
+                                  :short-name "coll1"
+                                  :native-id "coll1"
+                                  :provider-id "PROV1"})
+
+        coll2 (u/save-collection {:entry-title "coll2 entry title"
+                                  :short-name "coll2"
+                                  :native-id "coll2"
+                                  :provider-id "PROV1"})
+
+        coll3 (u/save-collection {:entry-title "coll3 entry title"
+                                  :short-name "coll3"
+                                  :native-id "coll3"
+                                  :provider-id "PROV1"})
+
+        coll4 (u/save-collection {:entry-title "coll4 entry title"
+                                  :short-name "coll4"
+                                  :native-id "coll4"
+                                  :provider-id "PROV1"})
+
+        acl1 (ac/create-acl (u/conn-context)
```

Comment:
There are a lot of ACL tests that do the acl1 acl2 ... acln thing.  I agree with you but this has been baked in for a while in this app.